apps/public-web/src/EShopOnAbp.PublicWeb/Pages/Index.cshtml

```diff
@@ -1,9 +1,12 @@
 ﻿@page
 @using EShopOnAbp.Localization
+@using EShopOnAbp.PublicWeb.Options
 @using Microsoft.Extensions.Localization
 @using System.Globalization
+@using Microsoft.Extensions.Options
 @model EShopOnAbp.PublicWeb.Pages.IndexModel
 @inject IStringLocalizer<EShopOnAbpResource> L
+@inject IOptions<RemoteServices> options
```

Comment:
See how we get Catalog RemoteService url in BasketService: https://github.com/abpframework/eShopOnAbp/blob/2af96a42cca83d987f7a63de2f1a2e147ad5b0d1/services/basket/src/EShopOnAbp.BasketService/BasketServiceModule.cs#L147-L148